pkgs/top-level/all-packages.nix

```
@@ -20075,16 +20089,27 @@ with pkgs;
     qt5 = qt515;
   });
 
+  libsForQt63 = recurseIntoAttrs (import ./qt6-packages.nix {
```

It's an unfortunate historical accident that we have `libsForQt5`, but `pythonPackages`, `perlPackages`, `haskellPackages`, etc. We should take the chance to correct this for Qt 6!

```suggestion
  qt6Packages = recurseIntoAttrs (import ./qt6-packages.nix {
```

---

+1

maybe add a (temporary) alias?

```nix
{
  libsForQt6 =
    builtins.trace "warning: `pkgs.libsForQt6' has been renamed to `pkgs.qt6Packages'."
    qt6Packages;
```